packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -283,6 +283,7 @@ gulp.task('icons-font', ['icons-common'], () => {
        */
       normalize: true,
       fontHeight: 1001,
+      timestamp: 1436442578, // Thu Jul 9 12:49:38 2015 GMT+1
```

Comment:
Haha love it. I reckon you'll need to put a better comment than that tho.

Perhaps: `A static timestamp to prevent changes showing up in git`

Reply:
fair enough 😄